src/repr/src/scalar.rs

```diff
@@ -168,6 +168,50 @@ pub enum Datum<'a> {
     // other datum variants should appear before `Null`.
 }
 
+impl Debug for Datum<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use mz_ore::str::*;
+        match self {
+            Datum::False => write!(f, "False"),
+            Datum::True => write!(f, "True"),
```

Comment:
Well, maybe we should redact even these? I can imagine certain columns whose True or False value is a sensitive thing. I'm not sure.

Reply:
Ok, then let's merge the current PR version for now!